mbedtls/src/error.rs

```diff
@@ -113,7 +113,6 @@ error_enum!(
         AriaFeatureUnavailable = ERR_ARIA_FEATURE_UNAVAILABLE,
         AriaHwAccelFailed = ERR_ARIA_HW_ACCEL_FAILED,
         AriaInvalidInputLength = ERR_ARIA_INVALID_INPUT_LENGTH,
-        AriaInvalidKeyLength = ERR_ARIA_INVALID_KEY_LENGTH,
```

Comment:
I think you need to define `MBEDTLS_DEPRECATED_REMOVED`

Reply:
It does not matter because even if the macro is defined, mbedtls won't return these errors anymore.

Reply:
Ah I see. If we've never released a version then we haven't change the public API. So it's ok to remove now. Thanks for the explanation.